lib/project_types/extension/commands/create.rb

```diff
@@ -6,6 +6,7 @@ class Create < ShopifyCli::SubCommand
       options do |parser, flags|
         parser.on('--name=NAME') { |name| flags[:name] = name }
         parser.on('--type=TYPE') { |type| flags[:type] = type.upcase }
+        parser.on('--api_key=KEY') { |key| flags[:api_key] = key.downcase }
       end
```

Comment:
```suggestion
        parser.on('--api_key=KEY') { |key| flags[:api_key] = key.downcase }
        parser.on('--api-key=KEY') { |key| flags[:api_key] = key.downcase }
      end
```

Adding this extra line supports both kebab-case and snake_case (following PR #1075 )

Reply:
As outlined in https://github.com/Shopify/shopify-app-cli/issues/1067#issuecomment-790970462, I don't believe that duplicating the flag configuration is necessary as the option parser supports both styles of arguments independent of how it was configured. Please see the code examples in the linked comment for details.

Reply:
I'm happy to change it to the more common kebab-case for improved readability even if the runtime behaviour is identical.

Reply:
@t6d I just saw that comment! That's really cool 😓 wish I had known that before putting in my PR duplicating everything